### ISSUE TITLE
Update README.Rmd

### ISSUE THREAD

**Issue by stefaniebutland** [Openscapes/kyber#116]
Update
- Openscapes Team
- ?rm "planning google folder"
- More about Openscapes and the Champions program:

**Comment by stefaniebutland** [comment]
Co-working -> Coworking

Better preamble for participating teams to know what to add

**Comment by stefaniebutland** [comment]
@jules32 to add text to underscore series more

**Comment by jules32** [comment]
Hi Stef! Could you please update this sentence

> To learn more about what to expect (the purpose, outcomes, and process of our Cohort), please read the homepage of the [Champions Program Lesson Series](https://openscapes.github.io/series). 

as: 

> To learn more about the program, what to expect, and outcomes, and lessons, please see the [Champions Program Lesson Series](https://openscapes.github.io/series). 

Feel free to improve but that was what was first on my mind. Thank you!!!

**Comment by stefaniebutland** [comment]
Is this the file I edit to update a repo's README template? https://github.com/Openscapes/kyber/blob/e6312f283c4b9c48cffb3f37a54160ae68b3bd0c/inst/rmarkdown/templates/openscapes-cohort-readme/skeleton/skeleton.Rmd

I also see this one that @jules32 was last to commit to: https://github.com/Openscapes/kyber/blob/main/inst/readmes/readme-template.Rmd

**Comment by stefaniebutland** [comment]
I see in [this commit](https://github.com/nmfs-openscapes/2024-nmfs-champions/commit/86c456bd6d03920e65f06fa5c8d95f08a72e1696) that kyber uses the https://github.com/Openscapes/kyber/blob/e6312f283c4b9c48cffb3f37a54160ae68b3bd0c/inst/rmarkdown/templates/openscapes-cohort-readme/skeleton/skeleton.Rmd REAMDE, which has less detail than Julie's version https://github.com/Openscapes/kyber/blob/main/inst/readmes/readme-template.Rmd

Since I'm updating README template, I'll add all to `skeleton.Rmd`

**Comment by stefaniebutland** [comment]
addressing this issue in #150 

documented need to use `skeleton.Rmd` in [kyber/README](https://github.com/Openscapes/kyber/blob/main/inst/rmarkdown/templates/openscapes-cohort-readme/skeleton/skeleton.Rmd)

**Comment by ateucher** [comment]
Don't know if this needs to happen in #150, but should the cohort parameter in the yaml frontmatter default to the repo name, so we don't have to change it manually?

**Comment by stefaniebutland** [comment]
That would be great, cause I keep getting an error when I forget. Would you mind doing that in another PR, or I could do it via screenshare (not today) and you tell me what to do?

**Comment by ateucher** [comment]
Yep will do! I haven't really looked at how the template is used/copied so don't know 100% yet the best way to implement it.

**Comment by stefaniebutland** [comment]
Completed in #150 

New issue opened #152
